docs/install/from_source.rst

```diff
@@ -107,6 +107,14 @@ The configuration of TVM can be modified by `config.cmake`.
       cmake ..
       make -j4
 
+  - You can also use Ninja build system instead of Unix Makefiles. It is said that Ninja runs faster than Makefiles.
```

Comment:
It can be faster to build than using Makefiles

Reply:
Addressed.